proto/viam/app/v1/app.proto

```diff
@@ -121,6 +121,9 @@ service AppService {
   // Get a list of robots
   rpc ListRobots(ListRobotsRequest) returns (ListRobotsResponse);
 
+  // Get a list of robots by organization
+  rpc ListRobotsByOrg(ListRobotsByOrgRequest) returns (ListRobotsByOrgResponse);
```

Comment:
Will `ListAuthorizationsForUser` already include all the robots that have permissions set for that user? If so, then do we still need `ListRobotsByOrg`?

Reply:
`ListRobotsByOrg` would still be useful for the select resource modal (where we load all robots a user has read access on regardless of current authorizations), but it's not strictly necessary because we can call `ListRobots` on every location we load.

I'll let @piokasar make the call on this one

Reply:
im tempted to say no because of what Michael said about reducing the number of endpoints we expose, but it would definitely cut down on the number of calls we make. Is there anywhere in the future where we would have to have the robots without needing the locations that would result in us using ListRobotsByOrg again?

Reply:
i don't think so unless we add more robot views on the org settings page, but i don't see an immediate reason for that. i'm good with removing it